src/branding/TerraHexagonsAnimation.ts

```diff
@@ -40,7 +40,6 @@ export const TerraHexagonsAnimation = (props: TerraHexagonsAnimationProps): Reac
         justifyContent: 'center',
         alignItems: 'center',
         height: size,
-        width: size,
```

Comment:
Leaving this in broke horizontal alignment of the hexagons; removing it fixed it and didn't seem to cause any problems.